sessions/react-global-state/demo-end/pages/overview.js

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components";
+import MainContent from "./components/MainContent";
+import Navigation from "./components/Navigation";
+
+const StyledNav = styled.nav`
+  padding: 1rem;
+`;
+
+const StyledMain = styled.main`
+  padding: 1rem;
+`;
+
+export default function OverviewPage({ animals, handleAdd, handleSubtract }) {
```

Comment:
Maybe `counter.js` is better name for the page

```suggestion
export default function CountersPage({ animals, handleAdd, handleSubtract }) {
```